coalib/parsing/ConfParser.py

```diff
@@ -156,7 +162,16 @@ def __parse_lines(self, lines, origin):
                                     remove_empty_iter_elements=
                                     self.__remove_empty_iter_elements),
                                     # Stop ignoring
-                            allow_appending=(keys == []))
+                            allow_appending=(
+                                keys == [] or keys == multiple_ignore_keys))
+            for key in current_section:
+                if key == 'ignore':
+                    del_newline = current_section.contents[key]._value
+                    # Remove unneccessary whitespaces
+                    del_newline = del_newline.replace(' ', '')
```

Comment:
this doesnt "del newline"  - this is killing all whitespace.

What about windows filenames which have spaces in them?

Stop doing string manipulation to hide the problems.

Build better test cases which describe reality.

Reply:
@jayvdb Do you want me to figure out something else for `del_newline = del_newline.replace(' ', '')` or is it that I change all the three operations corresponding to line no.s 171-173. And about the test cases, do you want me to make more of them?

Reply:
Should I find the component that already does string manipulation and use it in some way rather than writing explicit code for that?

Reply:
> Do you want me to figure out something else for del_newline = 

no. you must get rid of this chunk of code. 

no voodoo post processing string manipulation.

The Setting must be given the raw value.

> And about the test cases, do you want me to make more of them?

yes, more test cases.
yes, more detail in the test cases.
They will need to be more complicated, to show reality.

> Should I find the component that already does string manipulation

I've already shown you were it is.  You need to use this in your test cases:

https://github.com/coala/coala/pull/5815#discussion_r226865556

`Setting.__iter__` does the splitting and whitespace stripping.